### DISCUSSION TITLE
How to debug ?

### DISCUSSION THREAD

**Discussion by yellowbean** [reflex-dev/reflex#2925] [Q&A]
The app runs OK:
![Screenshot 2024-03-26 at 13 35 16](https://github.com/reflex-dev/reflex/assets/1008321/cb1e363a-d6ba-42b0-b31a-22072797becb)

But the browser seems not working : ( the red wifi sign )

Is there a way for me to debug it ?  Thanks !

![Screenshot 2024-03-26 at 13 36 30](https://github.com/reflex-dev/reflex/assets/1008321/8a49d259-30e6-4f39-aeda-241411d77a90)

**Comment by picklelo** [comment]
It looks like the backend may not be starting correctly. Is it accessible at `localhost:8000/ping`? You can also try to `reflex run --loglevel debug` and see if there's any errors in the console.
